packages/react/src/components/Modal/Modal.test.tsx

```diff
@@ -111,4 +111,24 @@ describe('Modal', () => {
     await render({ children, open: true });
     expect(screen.getByText(children)).toBeInTheDocument();
   });
+
+  /* We can't test the close function, but once vitest supports it, we can */
```

Comment:
Is there an issue we can link to here?

Reply:
I'll see if I can find one

Reply:
I went ahead and updated our mock, so it now works